tests/python_tests/test_whisper_generate_api.py

```diff
@@ -67,9 +67,7 @@ def read_whisper_model(params, **tokenizer_kwargs):
         model_id,
         path,
         opt_pipe,
-        ov_genai.WhisperPipeline(
-            path, 'CPU', config={'ENABLE_MMAP': False}
-        ),
+        ov_genai.WhisperPipeline(path, "CPU", config={"ENABLE_MMAP": False}),
```

Comment:
after this PR https://github.com/openvinotoolkit/openvino.genai/pull/1042 being merged, we don't have `config`